index.html

```diff
@@ -972,21 +958,13 @@ <h2>
       <p>
         The <a data-cite=
         "#dom-performancenavigationtiming">PerformanceNavigationTiming</a>
-        interface exposes timing information for the <a>current document</a> to
-        any resource loaded by the document, such as a web page or a worker. To
-        limit the access to the <a data-cite=
+        interface exposes timing information about the previous document to the
+        <a>current document</a>.
+        To limit the access to <a data-cite=
         "#dom-performancenavigationtiming">PerformanceNavigationTiming</a>
-        interface, the <a data-cite=
-        "resource-timing-2#dfn-timing-allow-check">timing allow check</a>
-        algorithm is enforced and certain attributes are set to zero, as
-        described in <a data-cite=
-        "resource-timing-2#sec-cross-origin-resources" class=
-        "sec-ref"><span class="secno">4.5</span> <span class=
-        "sec-title">Cross-origin Resources</span></a> [[RESOURCE-TIMING]].
-        Resource providers can explicitly allow all timing information to be
-        collected for a <a>current document</a> by adding the
-        <code>Timing-Allow-Origin</code> HTTP response header, which specifies
-        the domains that are allowed to access the timing information.
+        attributes which include information on the previous document, the
+        <a>same-origin check</a> algorithm is enforced and attributes related
+        to the current document are set to zero.
```

Comment:
```suggestion
        to the previous document are set to zero.
```

It's the ones related to the previous document that are set to zero.